recipes/libs/mesa3d.yaml

```diff
@@ -115,7 +115,7 @@ multiPackage:
                   - graphics::wayland::wayland-protocols
                   - graphics::wayland::wayland-dev
 
-            - if: "$(eq,$MESA3D_PLATFORM,x11)"
+            - if: "$(eq,$MESA3D_PLATFORM,wayland)"
```

Comment:
I'm confused. Now, if `$MESA3D_PLATFORM` equals `wayland`, we add the X11 dependencies. Which contradicts the commit message. What am I missing?

Reply:
:facepalm: our recipes are based on an older version of the basement where we can not use the `default` attribute in the `Config` section. That's why we had `if: "$(eq,${MESA3D_PLATFORM:-x11},x11)` here. ATM I'm upstreaming my changes in order to rebase later and obviously I wasn't very focused yesterday... Sorry for the noise..

Reply:
No worries!